runbotics/runbotics-scheduler/package.json

```diff
@@ -19,7 +19,7 @@
         "test:watch": "vitest dev",
         "test:cov": "vitest run --coverage",
         "test:debug": "vitest --inspect-brk --threads false",
-        "typeorm": "rm -rf ./dist && rushx build && ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js -d ./src/ormconfig.ts",
+        "typeorm": "rm -rf ./dist && rushx build && ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js -d ./dist/src/ormconfig.js",
```

Comment:
Maybe that's the reason reverting migrations stopped working in my local environment